app/models/course/assessment/submission.rb

```diff
@@ -135,6 +138,16 @@ def publish(_ = nil)
     end
   end
 
+  # Handles the unsubmission of a submitted submission.
+  #
+  # This will revert all manually graded answers to an attempting state.
```

Comment:
If that's the case I think it's better to `unsubmit` all answers, and make the student submit everything again. Between:
1. Having to unsubmit the submission then tutor unsubmitting the necessary answers; vs
2. Unsubmit the submissions then student re-submitted all answers

Option 2 seems more feasible. 

Thoughts @kxmbrian ?